internal/appconfig/config.go

```diff
@@ -190,14 +191,11 @@ func (cfg *Config) BuildStrategies() []string {
 	return strategies
 }
 
-func (c *Config) Volumes() []Volume {
+func (c *Config) Volumes() []Mount {
```

Comment:
Should this be called `Mounts` now?

Reply:
I'm afraid it can't because there is a field with the same name.

it could be `GetMounts` but not sure we need a version that reads volumes from rawDefinition. 

What was the motivation to add it?

Reply:
The motivation was the fact that Volumes (now Mounts) wasn't an array and didn't have the processes field in the struct definition.
It should be fine to get rid of, actually, yeah

Reply:
@alichay do you mind if we leave that to a follow up PR? I'd like to merge this before it diverges even more

Reply:
yeah no problem

Reply:
thanks